README.md

```diff
@@ -98,3 +127,13 @@ aglio -i docs/auth.apib -o docs/auth.html
 # serve apis locally
 aglio -i docs/auth.apib -s
 ```
+
+## Tests
+
+Auth have some automated test scripts.
+We use [pyresttest](https://github.com/svanoort/pyresttest) format for the test schemas.
```

Comment:
I didn't know this - it seems great! The test descriptions are really easy to understand and (it seems) to write.
Also, it looks like the sort of sanity test one could easily run within travis, to validate builds.